core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -815,9 +816,25 @@ public void removeBlocks(Collection<Long> blockIds, boolean delete) throws Unava
   }
 
   @Override
-  public void decommissionWorker(long workerId)
-      throws Exception {
-    //TODO(Tony Sun): added in another pr.
+  public boolean isServing(long workerId) {
```

Comment:
```suggestion
  public boolean isServing(long workerId) {
```

I have a small question here. If the function of `isServing` is judge whether target worker is decommissioned, if it is decommissioned, return `True`, so why not called the function `isNotServing` or just `isDecommissioned`? I remember if a worker is decommissioned, it will not supply any services.

Reply:
Make sense, thanks for pointing this!

Reply:
@jiacheliu3 Rename the `isServing` method to `isNotServing`, FYI.